### ISSUE TITLE
Canada.ca theme: using aria-required on label element?

### ISSUE THREAD

**Issue by awkawk** [wet-boew/GCWeb#1067]
Noticed the use of aria-required on the label element in the feedback form example.  As aria-required is correctly used on the input/select elements the required state is correctly conveyed.  aria-required belongs on the control, not the label.  This is not to suggest that the text "required" shouldn't be present - it should be - just that the aria-required attribute isn't useful on the label.

**Comment by awkawk** [comment]
Found the page at https://github.com/wet-boew/GCWeb/blob/master/site/pages/feedback-en.hbs - both the en and fr version have already fixed this, so I guess the issue is that the live example page isn't updated to reflect the change yet.

**Comment by hsrudnicki** [comment]
Can be this issue closed as resolved or otherwise?

**Comment by zzucco** [comment]
I have been unable to find any instances of the aria-required tag as specified. This issue has probably been fixed.

Unless some new information comes in, this issue can be closed.

**Comment by awkawk** [comment]
"Can't find anymore even after looking carefully" is pretty close to "fixed" :)

**Comment by nschonni** [comment]
Probably resolved by the bump to jQuery validation which removed the aria-required injection on regular `required` fields. This was impacted because of the `required` css class being used (maybe incorrectly) on the labels interacting with the old feature.
